scripts/us_census/acs5yr/subject_tables/common/common_util.py

```diff
@@ -0,0 +1,502 @@
+                                           column_name.split(delimiter)):
+                ret_value = True
+    return ret_value
+
+
+def remove_columns_to_be_ignored(column_name_list: list,
+                                 spec_dict: dict,
+                                 delimiter: str = '!!') -> list:
+    """Function that removes columns to be ignored from a given list of columns.
+
+    Args:
+      column_name_list: The list of column name strings.
+      spec_dict: Dict obj containing configurations for the import.
+      delimiter: delimiter seperating tokens within single column name string.
+    
+    Returns:
+      A list of filtered column names, with the column names to be ignored
+        removed from the input list.
+  """
+    ret_list = []
+    for column_name in column_name_list:
+        if not column_to_be_ignored(column_name, spec_dict, delimiter):
+            ret_list.append(column_name)
+    return ret_list
+
+
+def ignored_columns(column_name_list: list,
+                    spec_dict: dict,
+                    delimiter: str = '!!') -> list:
+    """Function that returns list of columns to be ignored from a given list of columns.
+
+    Args:
+      column_name_list: The list of column name strings.
+      spec_dict: Dict obj containing configurations for the import.
+      delimiter: delimiter seperating tokens within single column name string.
+    
+    Returns:
+      A list of column names that will be ignored according to the spec_dict.
+  """
+
+    ret_list = []
+    for column_name in column_name_list:
+        if column_to_be_ignored(column_name, spec_dict, delimiter):
+            ret_list.append(column_name)
+    return ret_list
+
+
+def get_tokens_list_from_column_list(column_name_list: list,
+                                     delimiter: str = '!!') -> list:
+    """Function that returns list of tokens present in the list of column names.
+
+    Args:
+      column_name_list: The list of column name strings.
+      delimiter: delimiter seperating tokens within single column name string.
+    
+    Returns:
+      A list of tokens present in the list of column names.
+  """
+
+    tokens = []
+    for column_name in column_name_list:
+        for tok in column_name.split(delimiter):
+            if tok not in tokens:
+                tokens.append(tok)
+    return tokens
+
+
+def get_spec_token_list(spec_dict: dict, delimiter: str = '!!') -> dict:
+    """Function that returns list of tokens present in the import configuration spec.
+
+    Args:
+      spec_dict: Dict obj containing configurations for the import.
+      delimiter: delimiter seperating tokens within single column name string.
+    
+    Returns:
+      A dict containing 2 key values:
+        token_list: list of tokens present in the spec_dict.
+        repeated_list: list of tokens that appear multiple times within the spec.
+  """
+    ret_list = []
+    repeated_list = []
+    # check if the token appears in any of the pvs
```

Comment:
`# check` in comment but no check return tru/false. perhaps reword to '# collect tokens that appear in any of the pvs`?